docs/source/getting-started/content/proc_starting-the-virtual-machine.adoc

```diff
@@ -38,5 +38,5 @@ $ {bin} start --vm-driver virtualbox --bundle _path_to_system_bundle_
 [NOTE]
 ====
 * The cluster takes a minimum of four minutes to start the necessary containers and operators before serving a request.
-* If errors are encountered on the start command due to a mismatch in the hyperkit bundle, use the '{bin} delete' command to remove older versions.  These older versions may have user data and care should be taken before issuing the command to insure past work is not lost. 
+* If errors are encountered on the start command due to a mismatch in dependencies (such as the hyperkit bundle), use the '{bin} delete' command to remove older versions.  These older versions may have user data and care should be taken before issuing the command to insure past work is not lost. 
```

Comment:
I would put this in the troubleshooting section, where there is more space and a sample message can be added.
"dependencies" is not the right term. "A mismatch in bundle versions" maybe. I would not mention at all the hyperkit bundle.
You have an extra space before "These older". I believe it's "to ensure past work...", not "to insure"

Reply:
I've created a section in the Troubleshooting chapter and also added a Note in the crc setup section. Without a way to handle the data from previous versions, documenting this has spread across a number of places.
It might be better to document this in the "Known Issues" section instead, especially if this is a short term issue. This way it can still be found and doesn't intrude across the documentation. Please let me know what you prefer.